### PR TITLE
Adding basic pure functions and component setup

### PR DESCRIPTION
**Pull request by sanchay811**
- Set up initial pure functions for operations.
- Update package.json file to ensure dependency consistency.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,6 +28,8 @@
   },
   "dependencies": {
     "react": "16.6.3",
+    "react-bootstrap": "^0.32.4",
```
Comment (howardhenry) [inline package.json]: In apps, your should favour fixed dependency versions so the modules behave as intended after each install. Note: this is different when you're creating a module.

```suggestion
    "react-bootstrap": "0.32.4",
```

File: src/components/index.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+import * as serviceWorker from './serviceWorker';
```
Comment (howardhenry) [inline src/components/index.js]: `App` and `serviceWorker` don't exist in the project so they should be removed from the PR. What are their functionality?

File: package.json (hunk 1 above)
Comment (sanchay811) [reply package.json]: Yeah it makes sense to set them all to a specific version, will update that. Right, a custom module would grow in versioning one develops it.

File: src/components/index.js (hunk 2 above)
Comment (sanchay811) [reply src/components/index.js]: They need to be removed, they were in use before from a tutorial. Will update the PR.

**Comment by sanchay811** [comment]
You're right it shouldn't be a work in progress. To keep it concise as to what the intention for the PR is; I'll revert that commit and update the PR. 

Thanks Howard, looking forward to learning more.

**Comment by howardhenry** [comment]
👍🏽

**Pull request merged**